src/tiny_slam/tiny_fascade.h

```diff
@@ -9,20 +9,45 @@
 
 #include "tiny_world.h"
 
+/*!
+ * \brief Derived class from LaserScanObserver to manage all world
+ *
+ * Handles data came from laser scanner and locates views if he is set.
+ * Presents a pattern fascade for all algorithm "tiny slam"
+ *
+ */
 class TinySlamFascade : public LaserScanObserver {
 private:
   using ScanMatcherObsPtr = std::shared_ptr<GridScanMatcherObserver>;
 public: // methods
   // TODO: copy ctor, move ctor, dtor
+
+  /*!
+   * Parameterized constructor sets all data members
+   * \param[in] gcs           - Grid Cell Strategy - configuration of cells in map
+   * \param[in] params        - configuration of laser scan (sets the quality value)
+   * \param[in] skip_max_vals - boolean variable - flag shows how are high values in messages will be handeled
```

Comment:
"<...> the values that exceed the max one specific to laser scanner are handled"